doc/outdated/Build-Environment.md

```diff
@@ -73,12 +73,12 @@ The following functionality is provided:
 
 ### Configuration Part
 Configuration part, in whatever build system it is implemented, check the following system specifics:
-+ Platform check - part of standard check in Autotools and Cmake. Very easy to figure out in Scons;
++ Platform check - part of standard check in Autotools and Cmake. Very effortless to figure out in Scons;

```

Comment:
This does not look like idiomatic English to me (I am not a native speaker through). What is the theory behind replacing "easy"? It's one of the most frequent words in the language (https://www.sketchengine.eu/wp-content/uploads/word-list/english/english-word-list-total.pdf).

Reply:
As it is not a mandatory word to be addressed, I have reverted it to the original word, this word is going to be added in the exception list.